ipatests/test_integration/test_external_ca.py

```diff
@@ -83,7 +83,7 @@ def install_server_external_ca_step1(host, extra_args=()):
 def install_server_external_ca_step2(host, ipa_ca_cert, root_ca_cert,
                                      raiseonerr=True):
     """Step 2 to install the ipa server with external ca"""
-    args = ['ipa-server-install',
+    args = ['ipa-server-install', '-U', '-r', host.domain.realm,
             '-a', host.config.admin_password,
```

Comment:
Is this change related?

Reply:
Yes @rcritten,

I found out that it is the external CA second step that was  causing a timeout for PRCI (see so many re-runs). Because of that I added an unattended option to this task. Without it it got stuck at new NTP prompt and then timeouted.

However it may happen that more nightly tests which rely on a fact that if all necessary options were passes an unattended option is not needed.

I see missing unattended option here as well because test is passing stdin text:

https://github.com/freeipa/freeipa/blob/master/ipatests/test_integration/test_installation.py#L587-L572

for this one a stdin input needs to be updated to have a ntp ptompt stdin text passed to installer.